packages/mui-material/src/Popover/Popover.js

```diff
@@ -366,7 +362,40 @@ const Popover = React.forwardRef(function Popover(inProps, ref) {
 
   let transitionDuration = transitionDurationProp;
 
-  if (transitionDurationProp === 'auto' && !TransitionComponent.muiSupportAuto) {
+  const externalForwardedProps = {
+    slots: {
+      transition: TransitionComponent,
+      ...slots,
+    },
+    slotProps: {
+      transition: TransitionProps,
+      ...slotProps,
+      paper: externalPaperSlotProps,
```

Comment:
We can do:

```suggestion
      paper: PaperPropsProp,
      ...slotProps,
```

And remove `externalPaperSlotProps`

Reply:
Done.